### DISCUSSION TITLE
OGCS won't start automatically

### DISCUSSION THREAD

**Discussion by mjpederson61** [phw198/OutlookGoogleCalendarSync#1979] [Q&A]
I have OGCS 2.11 on a Windows 11 PC.  I recently reinstalled OGCS because Outlook was acting "funny" (would complete folder sync).  This fixed Outlook, but OGCS now won't start automatically.  It is set to start 90 seconds after bootup.  Thougths?  Does OGCS have to be set as a StartUp App in Windows?

**Comment by phw198** [comment]
Would need to see your log file at the point you toggle the option on for "start on login" really, but at a guess OGCS is unable to set the required registry key entry because it doesn't have permission.

You could check if your registry has an entry under the Current User for `Software\Microsoft\Windows\CurrentVersion\Run\Outlook Google Calendar Sync`...

**Reply by mjpederson61** [reply]
Checked registry and did find this entry under Current User.  Have tried toggling the "start on login" feature and also played with the time to start.  Also, I can see OGCS as an Installed app in Windows Settings, but not as an option to under Startup Apps.
